create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -81,6 +85,15 @@ void shouldExtractScopusIdentifierAndPlaceItInsideAdditionalIdentifiersObject()
         assertThat(actualAdditionalIdentifiers, contains(expectedIdentifier));
     }
 
+    @Test
+    void shouldExtractDoiAndPlaceItInsideReferenceObject() throws IOException, URISyntaxException {
+        var scopusFile = IoUtils.stringFromResources(Path.of(SCOPUS_XML_0000469852));
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+        S3Event s3Event = createS3Event(uri);
+        CreatePublicationRequest createPublicationRequest = scopusHandler.handleRequest(s3Event, CONTEXT);
+        assertThat(createPublicationRequest.getEntityDescription().getReference().getDoi(), equalToObject( new URI( DOI_OPEN_URL_FORMAT+ "/" +DOI_IN_0000469852)));
```

Comment:
here you should have the URI.create(<URI string>) so that we are sure that the URI is exactly what we are expecting

Reply:
Went for the UriWrapper as you suggested.